sources/install/package_osint.sh

```diff
@@ -414,6 +414,8 @@ function package_osint() {
     install_theharvester            # Gather emails, subdomains, hosts, employee names, open ports and banners
     install_h8mail                  # Email OSINT & Password breach hunting tool
     install_infoga                  # Gathering email accounts informations
+    pip3 install cython Cython
+    pip3 install cchardet
```

Comment:
Please opt for `python3 -m pip install` instead so that it's more explicit
Also, for what purpose are those libs installed? If it's for a specific tool, then they should be added to that specific tool's virtual env. All Python tools need to have their own venv (either with pipx or manually)